KIPs/kip-71.md

```diff
@@ -204,12 +204,23 @@ class World(ABC):
 		transactions = self.transactions(block)
 		parent_gas_used = self.parent(block).gas_used
 
+		lower_bound_base_fee = LOWER_BOUND_BASE_FEE
+		upper_bound_base_fee = UPPER_BOUND_BASE_FEE
+
+		# if LOWER_BOUND_BASE_FEE is not a even number, make it even by adding 1
+		if lower_bound_base_fee & 0x1:
+			lower_bound_base_fee += 1
+
+		# if UPPER_BOUND_BASE_FEE ix not a even number, make it even by subtracting 1
+		if upper_bound_base_fee & 0x1:
+			upper_bound_base_fee += 1
```

Comment:
for upper_bound_base_fee, we can use a subtraction(-1) strategy in order the nextBaseFee not to exceed it.

Reply:
Oh, yeap, you're right. I will update it.

Reply:
@kjeom Updated. Please take a look again. Thanks!

Reply:
Thanks!